packages/evm/src/index.ts

```diff
@@ -30,4 +35,14 @@ export type {
   PrecompileInput,
 }
 
-export { EOF, EVM, EvmError, EVMErrorMessage, getActivePrecompiles, getOpcodesForHF, Message }
+export {
+  EOF,
+  EVM,
+  EvmError,
+  EVMErrorMessage,
+  getActivePrecompiles,
+  getOpcodesForHF,
+  MCLBLS,
+  Message,
+  NobleBLS,
```

Comment:
I'm not sure if we want to export these semi-internal `NobleBLS` and `MCLBLS`?

Reply:
Nvm, I now see, we want this to init MCLBLS if users want to use it. (Read other comment for some more suggestions)